service/src/test/java/uk/gov/hmcts/reform/fpl/service/orders/validator/EPORemovalAddressValidatorTest.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.hmcts.reform.fpl.service.orders.validator;
+
+import org.assertj.core.api.AssertionsForClassTypes;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.fpl.enums.EPOType;
+import uk.gov.hmcts.reform.fpl.model.Address;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.event.ManageOrdersEventData;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.model.order.OrderQuestionBlock.EPO_PREVENT_REMOVAL;
+
+class EPORemovalAddressValidatorTest {
+
+    private static final String INVALID_ADDRESS_LINE = "Enter a valid address for the contact";
+    private static final String INVALID_POST_CODE = "Enter a postcode for the contact";
+
+    private final EPORemovalAddressValidator underTest = new EPORemovalAddressValidator();
+
+    @Test
+    void accept() {
+        AssertionsForClassTypes.assertThat(underTest.accept()).isEqualTo(EPO_PREVENT_REMOVAL);
```

Comment:
Although this class will work fine for assertions it is technically the wrong one. This class is for java 7 and lower and contains work arounds for features that were not imlpemented there but are in java 8+. You already statically import the other assertThat so this one can be removed.